slack_sdk/models/blocks/basic_components.py

```diff
@@ -559,3 +559,25 @@ def to_dict(self) -> Dict[str, Any]:  # skipcq: PYL-W0221
         else:
             json["trigger"] = self._trigger
         return json
+
+
+class SlackFile(JsonObject):
```

Comment:
Maybe add a docstring linking to https://api.slack.com/reference/block-kit/composition-objects#slack_file here?

Reply:
I've added the comment for this class!